File: 프로그래머스/0/120837. 개미 군단/개미 군단.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int hp) {
    int answer = 0;
    
    int a = hp / 5; // a 장군개미 수
    int b = hp % 5 / 3; // b 병정개미 수
    int c = hp % 5 % 3 / 1; // c 일개미 수
    
    answer = a + b + c;
    return answer;
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.js
function solution(hp) {
    var answer = 0;
    
    var first = 0; // 장군개미 (5)
    var second = 0; // 병정개미 (3)
    var third = 0; // 일개미 (1)
    
    first = parseInt(hp / 5);
    hp = parseInt(hp % 5);
    second = parseInt(hp / 3);
    third = parseInt(hp % 3);
    
    answer = first + second + third;
    return answer;
}